ops/delivery-e2e.cloudbuild.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+steps:
+  # Clean up any old Emblem instance cruft.
+  - name: '${_REGION}-docker.pkg.dev/${_PROJECT}/e2e-testing/deployer:${_E2E_DEPLOYER_SHA}'
+    entrypoint: /bin/bash
+    args:
+      - ./terraform/cleanup.sh
+    env:
+      - "EMBLEM_BILLING_ACCOUNT=${_EMBLEM_BILLING_ACCOUNT}"
+      - "EMBLEM_ORGANIZATION=${_EMBLEM_ORGANIZATION}"
+      - "PROJECT_NUMBER=$PROJECT_NUMBER"
```

Comment:
Not sure if it was lost, why is billing account and organization needed?

Reply:
Merge issue, I think.